driver-core/src/main/com/mongodb/client/model/expressions/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * API for MQL expressions.
+ *
+ * @see com.mongodb.client.model.expressions.Expression
+ */
+@NonNullApi
+package com.mongodb.client.model.expressions;
```

Comment:
Given that this package currently provides the very core of a new computational API that uses MQL behind the scenes, is the name `expressions` really a good fit for it? Maybe `mql` is a better package name, or something else?

Reply:
I was wondering how to rename it, and `mql` is a good idea.